assign_rights/assemble.py

```diff
@@ -21,3 +26,12 @@ def create_json(self):
     def return_rights(self):
         """docstring for return_rights"""
     pass
+
+    def run(self, request_list):
+        shell_list = []
+        for item in request_list:
+            try:
+                self.retrieve_rights(item, shell_list)
+            except Exception as e:
```

Comment:
As this method continues to get built out, we'll want to target specific exceptions, for example an id that doesn't match any RightsShell objects will raise a `RightsShell.NotFoundError` (I think). But that can wait until later.

Reply:
Right now if there is an error that gets sent: `Error retrieving rights shell: RightsShell matching query does not exist.`

Reply:
Right, but we might want to return a 404 if we can't find a rights shell, or a 500 if something else goes wrong. Exception handling is something that is usually built out incrementally though, so it's more something to keep in mind and less something to act on right now.